terraform/groups/ecs-service/locals.tf

```diff
@@ -0,0 +1,85 @@
+# Define all hardcoded local variable and local variables looked up from data resources
+locals {
+  stack_name                = "company-requests" # this must match the stack name the service deploys into
+  name_prefix               = "${local.stack_name}-${var.environment}"
+  service_name              = "promise-to-file-web"
+  container_port            = "3000" # default node port required here until prod docker container is built allowing port change via env var
+  docker_repo               = "promise-to-file-web"
+  lb_listener_rule_priority = 13
+  lb_listener_paths         = ["^/company-required/*"]
+  healthcheck_path          = "/company-required" #healthcheck path for promise to file web
+  healthcheck_matcher       = "200-302"           # no explicit healthcheck in this service yet, change this when added!
```

Comment:
This is incorrect now, should be the new healthcheck path, and the matcher can be 200 only